harness/helpers.ts

```diff
@@ -22,4 +22,25 @@ export const forEachSDK = (tests) => {
     describe.each(SDKs)('%s SDK tests', tests)
 }
 
+export const createClient = async (url: string, clientId: string, sdkKey: string, options?: object) => {
+    return await fetch(`${url}/${clientId}`, {
+        method: 'POST',
+        body: JSON.stringify({
+            clientId,
+            sdkKey,
+            options
+        })
+    })
+}
+
+export const createUser = async (url: string, user: object) => {
+    const response = await fetch(`${url}/user}`, {
+        method: 'POST',
+        body: JSON.stringify({
+            ...user
+        })
+    })
+    return response.text()
```

Comment:
any reason for not doing `.json()` instead?

Reply:
actually I think we should return the entire response object cause the tests will need to get the `Location` header